lms/resources/_js_config/__init__.py

```diff
@@ -290,70 +290,32 @@ def add_deep_linking_api(self):
         self._config.setdefault("filePicker", {})
         self._config["filePicker"]["deepLinkingAPI"] = config
 
-    def enable_instructor_toolbar(self, enable_editing=True, enable_grading=False):
-        """
-        Enable the toolbar with controls for instructors in LMS assignments.
+    def enable_toolbar_editing(self):
+        toolbar_config = self._get_toolbar_config()
 
-        :param enable_editing: Whether to enable controls for editing assignment configuration
-        :param enable_grading: Whether to enable grading controls
-        """
+        toolbar_config["editingEnabled"] = True
+        self._config["instructorToolbar"] = toolbar_config
```

Comment:
It would be nice to avoid the duplicated assignment in `enable_toolbar_editing` and `enable_toolbar_grading`. A possible approach would be to change `_get_toolbar_config` to be `_create_toolbar_config` and make that helper responsible for assigning the `instructorToolbar` key.